tests/Unit/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/Test_GaussianPlusConstant.cpp

```diff
@@ -0,0 +1,121 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cmath>
```

Comment:
I'll leave you to double check includes here. You shouldn't need any forward decls in a test file because everything needs to be `include`d